Sources/CvPlot.cpp

```diff
@@ -16456,50 +16145,40 @@ void CvPlot::setImprovementUpgrade()
 		{
 			if ((ImprovementTypes)GC.getBuildInfo((BuildTypes)iI).getImprovement() == eUpgrade)
 			{
-				BuildTypes eBuild = ((BuildTypes)iI);
```

Comment:
You shouldn't remove this, it's better than doing `(BuildTypes)iI` every time. It should be moved up a few lines, made cost, used in all the places.